sql/core/src/main/scala/org/apache/spark/sql/execution/aggregate/ObjectHashAggregateExec.scala

```diff
@@ -123,7 +123,7 @@ case class ObjectHashAggregateExec(
             resultExpressions,
             (expressions, inputSchema) =>
               MutableProjection.create(expressions, inputSchema),
-            child.output,
+            inputAttributes,
```

Comment:
There's actually another `child.output` at line 118, but it's dead code indeed.

Reply:
You can open a new PR to remove dead code

Reply:
ok..